realm-annotations-processor/src/main/java/io/realm/processor/RealmJsonTypeHelper.java

```diff
@@ -47,48 +48,53 @@
         JAVA_TO_JSON_TYPES.put("java.lang.String", new SimpleTypeConverter("String", "String"));
         JAVA_TO_JSON_TYPES.put("java.util.Date", new JsonToRealmTypeConverter() {
             @Override
-            public void emitTypeConversion(String setter, String fieldName, String fieldType, JavaWriter writer) throws IOException {
+            public void emitTypeConversion(String setter, String fieldName, String fieldType, JavaWriter writer)
+                    throws IOException {
                 writer
-                    .beginControlFlow("if (!json.isNull(\"%s\"))", fieldName)
+                        .beginControlFlow("if (!json.isNull(\"%s\"))", fieldName)
```

Comment:
Yes, and I think the code is more important than the generated code. You want to be able to tell the flow of the generated code at a glance which you cannot do anymore if this is changed. This has not been mentioned once in code review for the last 6 months, so I would expect we all agreed that it was more readable?